.env.example

```diff
@@ -1,6 +1,8 @@
 URL=http://localhost:3463
 DATO_API_TOKEN=
 MAPBOX_TOKEN=
+SUPABASE_URL="https://pfsmhuunrlzvdbteqfku.supabase.co"
+SUPABASE_KEY=""
```

Comment:
Change this to `SUPABASE_ANON_KEY`? To prevent confusion on which key should be used here